weekly-mission/next-project/src/components/Button/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ButtonHTMLAttributes } from "react";
+import styles from "./Button.module.css";
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  btnType?: string;
+}
+
+const Button = ({ children, btnType, onClick }: ButtonProps) => {
+  const buttonClass = btnType ? styles[btnType] : "";
+  return (
+    <button
+      className={`${styles.StyledButton} ${buttonClass}`}
+      onClick={onClick}
+      color-type={btnType}
+    >
```

Comment:
ButtonHTMLAttributes<HTMLButtonElement> 을 쓰신만큼 바로 활용해주셔도 좋을 거 같아요!
```suggestion
  const Button = ({ children, btnType, onClick, ...rest }: ButtonProps) => {
  ...
    <button
      className={`${styles.StyledButton} ${buttonClass}`}
      onClick={onClick}
      color-type={btnType}
      ...rest
    >
```